### PR TITLE
[MIG] Ubuntu 22.04 changes

### PR DESCRIPTION
**Pull request by quentinDupont**
Changes for Ubuntu 22.04 version
Improve install.sh 
- indempotent
- option for https or not
- upgrade mkcert version to 1.4.4 to make it work on Firefox again

Tested on two new Ubuntu 22.04 computers, for Epson TMT20-III, customer display and one payment device

### CODE REVIEW

**Comment by PierrickBrun** [comment]
quick code review: looks good

I hope it will not break any backward compatibility but I'm not sure about it.

If 18.04 breaksI think this is not a problem but if 18 and 20 breaks we may have to create a "focal" branch to maintain it

**Comment by rvalyi** [comment]
that being said, a rebase will probably be required as the pre-commit cleanup will probably conflict with https://github.com/akretion/pywebdriver/pull/109 (not sure why the merge is still running BTW).

**Review by rvalyi — APPROVED** [review]
LGTM (disclaimer I'm not an expert in Debian packages).

**Comment by quentinDupont** [comment]
build.sh for ubuntu 22 has done the job well. Stil need to test it with real printer etc. Coming soon

**Comment by quentinDupont** [comment]
Changes for Ubuntu 22.04 version :heavy_check_mark: 

Build.sh functional :heavy_check_mark: 

Improve also install.sh 
- indempotent
- option for https or not
- upgrade mkcert version to 1.4.4 to make it work on Firefox again


Tested on two new Ubuntu 22.04 computers, for Epson TMT20-III, customer display and one payment device

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ repos:
         args:
           - --settings=.
         exclude: /__init__\.py$
-  - repo: https://gitlab.com/PyCQA/flake8
+  - repo: https://github.com/PyCQA/flake8
```
Comment (rvalyi) [inline .pre-commit-config.yaml:70]: le commit de connaisseur!  ;-)

**Review by rvalyi — CHANGES_REQUESTED** [review]
LGTM but without test nor extreme care in the review on my side.

**Review by rvalyi — APPROVED** [review]
LGTM but without test nor extreme care in the review on my side.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (quentinDupont) [reply .pre-commit-config.yaml:70]: je dois tout à @legalsylvain qui a fait ce commit dans nos repos ;) j'avais pas compris sur le moment, j'ai compris quand ça a pété ici !

**Comment by quentinDupont** [comment]
How can we make it go forward ? :) 

At the moment, to be tested, to install it, you need to :
- compile or download new .deb here https://nuage.grap.coop/s/YRetJsiiPiZpjn9 (waiting github actions when it will be merged)
- download install.sh https://github.com/akretion/pywebdriver/blob/554cca7110e0d3634248d9a26682382f3b9d54b6/debian/install.sh
- make it executable
- change lines to install .deb that we just downloaded
- ./install.sh
- change lines in /etc/pywebdriver/config.ini to add
sslcert=/etc/pywebdriver/cert/localhost+1.pem
sslkey=/etc/pywebdriver/cert/localhost+1-key.pem
(it will be automatic but config file is changed is this PR)
- restart pywebdriver ` sudo service pywebdriver restart`
- Check in https://localhost:8069 to accept certificate
- it should be all good

**Comment by quentinDupont** [comment]
Thank youu @PierrickBrun !!

**Pull request merged**